libraries/ashrae/guideline36/4.11-hot-water-plant.ttl

```diff
@@ -0,0 +1,446 @@
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:Valve_Status ] ;
+        sh:qualifiedMinCount 0 ;
+        sh:qualifiedMaxCount 1 ;
+    ] .
+
+# TODO see comment
+:isolation-valve a sh:NodeShape, owl:Class, bmotif:HVAC ;
+    rdfs:label "Isolation Valve" ;
+    rdfs:comment "Required for boiler plants with headered primary pumps. Delete otherwise." ;
+    sh:class brick:Isolation_Valve ;
+    # applications, required if present
+    sh:property [
+        rdfs:label "Boiler isolation valve" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:On_Off_Command ] ;
+        sh:qualifiedMinCount 1 ;
+        sh:qualifiedMinCount 1 ;
+    ] ;
+    # Retain the following two points for optional valve end switch status feedback.
+    # optional
+    sh:property [
+        rdfs:label "Boiler isolation valve closed end switch" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:Switch_Status ] ;
+        sh:qualifiedMinCount 0 ;
+        sh:qualifiedMaxCount 1 ;
+    ] ;
+    # optional
+    sh:property [
+        rdfs:label "Boiler isolation valve open end switch" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:Switch_Status ] ;
+        sh:qualifiedMinCount 0 ;
+        sh:qualifiedMaxCount 1 ;
+    ] .
+
+# 4.11.2 HW Pumps (see components.ttl)
+
+# 4.11.3 Primary (only) HW Loop
+# NOTE this subsection mixes primary-only points with primary-secondary points.
+# Thus, these two subsections have been explicitly separated into
+# Primary (only) HW Loops and (Primary) Secondary HW Loops in 4.11.4.
+
+:primary-only-loop a sh:NodeShape, owl:Class, bmotif:HVAC ;
+    sh:class brick:Hot_Water_Loop ;
+    # applications, TODO see comment
+    sh:property [
+        rdfs:label "HW supply or return flow" ;
+        rdfs:comment "HW flow required for primary-only plants; the flow meter must be located on the plant side of the HW minimum flow bypass where provided." ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [
+            sh:or ( sh:class brick:Leaving_Hot_Water_Flow_Sensor sh:class brick:Entering_Hot_Water_Flow_Sensor )
+        ] ;
+        sh:qualifiedMinCount 1 ;
+        sh:qualifiedMaxCount 1 ;
+    ] ;
+    # HW Min flow bypass valve see variable-primary-only-loop
+    # applications
+    sh:property [
+        rdfs:label "HW supply temperature" ;
+        rdfs:comment "HW supply temperature sensor required for primary-only plants." ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:Leaving_Hot_Water_Temperature_Sensor ] ;
+        sh:qualifiedMinCount 1 ;
+        sh:qualifiedMaxCount 1 ;
+    ] ;
+    # Plant HW return temperature see variable-primary-only-loop
+    # Loop HW return temperature see variable-primary-only-loop
+    # optional, TODO https://github.com/BrickSchema/Brick/issues/532
+    sh:property [
+        rdfs:label "HW system gauge pressure" ;
```

Comment:
@MatthewSteen 
1. Variable_Frequency_Drive unit of measure is HZ but for the VFD speed is %
2. Fault and Alarm are almost the same thing, the only difference is Alarms are triggered and give a notification unlike Faults
3. Chiller Fault Code is AI point but [Last_Fault_Code_Status](https://brickschema.org/ontology/Nightly/classes/Last_Fault_Code_Status) is DI point 
4. I don’t think Pump_Command/Fan_Command is an analog point. It is DO/BO point. It will be an AO point when it is Pump_Speed_Command/Fan_Speed_Commad
5. Usually we use open/Close enumeration for the Valve and damper, and on/off is for the Fan and pumps. And the Brick https://brickschema.org/ontology/1.3/classes/Open_Close_Status will show the current valve status (open/closed)
6. https://brickschema.org/ontology/1.3/classes/Valve_Command can be either DO/AO. Not common to use On_Off_Command for Dampers and Valves
7. I agree with your comment “WSEs are HXs and the Economizer definition is vague”. Still, we call a damper an Economizer in the air-side economizer system, right?